beta/docker/README.md

```diff
@@ -174,87 +174,66 @@ docker stack config \
 
 ## Test your SCIM bridge
 
-Run this command to retrieve logs from the service for the SCIM bridge container:
+Run this command to view logs from the service for the SCIM Bridge container:
 
 ```sh
 docker service logs op-scim-bridge_scim --raw
 ```
 
-Your SCIM bridge URL is based on the fully qualified domain name of the DNS record created in [Get started](#get-started), for example `https://scim.example.com/`. You can access your SCIM bridge in a web browser at this URL by signing in using your bearer token.
-
-You can also test your SCIM bridge by sending an authenticated SCIM API request.
-
-_Example command:_
+Your **SCIM Bridge URL** is based on the fully qualified domain name of the DNS record created in [Get started](#get-started), for example `https://scim.example.com`. Replace `mF_9.B5f-4.1JqM` with your bearer token and `https://scim.example.com` with your SCIM Bridge URL to test the connection and view status information.
```

Comment:
This new edit doesn't quite have enough context on what the command below does. It's another way of getting logs, right? We should probably say that. Here are some style edits as well:

```suggestion
Your **SCIM bridge URL** is based on the fully qualified domain name of the DNS record created in [Get started](#get-started). For example: `https://scim.example.com`. Replace `mF_9.B5f-4.1JqM` with your bearer token and `https://scim.example.com` with your SCIM bridge URL to test the connection and view status information.
```

Reply:
I am expecting that

> to test the connection and view status information.

gives the needed context, but that's why I'm asking for your review! 😉

It does not return logs. The example response following this shows what should be expected, which is a JSON representation of the data that is presented visually when signing in to the SCIM Bridge URL in a web browser: health status of the components of SCIM bridge.

I would assume that any error responses should be self-evident for the most part.

Reply:
I think this is the final unresolved thread, @Penderworth.

Reply:
Ah okay, this is good then! Just apply the suggestion above for style edits.